packages/eslint-plugin-svelte/src/shared/svelte-compile-warns/transform/typescript.ts

```diff
@@ -35,6 +35,7 @@ export function transform(
 					ts.ScriptTarget.ESNext,
 				module: ts.ModuleKind.ESNext,
 				importsNotUsedAsValues: ts.ImportsNotUsedAsValues.Preserve,
+				preserveValueImports: true,
```

Comment:
Thank you for your comment!
I knew it was deprecated, but I didn't know it would be removed in 5.5. It seems that we need to specify both so that it works with older versions of typescript 😅